recipes/glog/all/conanfile.py

```diff
@@ -37,31 +42,36 @@ def _source_subfolder(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if self.settings.os not in ["Linux", "FreeBSD"]:
+            del self.options.with_unwind
 
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
         if self.options.with_gflags:
             self.options["gflags"].shared = self.options.shared
+            self.options["libunwind"].shared = self.options.shared
 
     def requirements(self):
         if self.options.with_gflags:
             self.requires("gflags/2.2.2")
+        if self.options.get_safe("with_unwind"):
+            self.requires("libunwind/1.6.2")
 
     def build_requirements(self):
-        if tools.Version(self.version) >= "0.6.0":
+        if Version(self.version) >= "0.6.0":
             self.build_requires("cmake/3.22.3")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
+        get(**self.conan_data["sources"][self.version],
                   destination=self._source_subfolder, strip_root=True)
 
     def _patch_sources(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
-            tools.patch(**patch)
+            patch(**patch)
```

Comment:
There is `apply_conandata_patches()` which is more elegant.

Reply:
yes, it was my first try for the conanv2 migration. It should be better now